docs/dev/cli/saucectl/artifacts/download.md

```diff
@@ -10,9 +10,8 @@ Downloads artifacts for a job from Sauce Labs.
 
 ## Usage
 
-<span className="cli">$ saucectl artifacts download jobID globPattern [flags]</span>
+`$ saucectl artifacts download jobID globPattern [flags]`
```

Comment:
What do you think about replacing the inline code formatting with a code block? That way the command can be copied and pasted.

Reply:
I had considered that as well. @tianfeng92  which do you prefer?

Reply:
That looks perfect!

Reply:
I have updated all of the Usage sections in the command references to use a code block in saucectl and virtual-usb.